renderer-macos/lib/bindings/BriskView.re

```diff
@@ -47,3 +50,34 @@ external setBackgroundColor:
   ) =>
   unit =
   "ml_BriskView_setBackgroundColor_bc" "ml_BriskView_setBackgroundColor";
+
+[@noalloc]
+external setShadowOffset: (t, [@unboxed] float, [@unboxed] float) => unit =
+  "ml_BriskView_setShadowOffset_bc" "ml_BriskView_setShadowOffset";
+
+[@noalloc]
+external setShadowRadius: (t, [@unboxed] float) => unit =
+  "ml_BriskView_setShadowRadius_bc" "ml_BriskView_setShadowRadius";
+
+[@noalloc]
+external setShadowOpacity: (t, [@unboxed] float) => unit =
+  "ml_BriskView_setShadowOpacity_bc" "ml_BriskView_setShadowOpacity";
+
+[@noalloc]
+external setShadowColor:
+  (
+    t,
+    [@unboxed] float,
```

Comment:
Would it be possible to label these parameters? It would make this easier to use.

Reply:
I'm not sure, I try not to mess with `external`s. 😅
We could always shadow it, couldn't we?

Reply:
True, shadowing is possible. Was just curious because we can label parameters in BuckleScript externals. I don't know much about the C FFI :-)

Reply:
Apparently, we can have labeled arguments after all. I will take a look.